libwget/html_url.c

```diff
@@ -66,8 +66,8 @@ static const char maybe[256] = {
 	['u'] = 1,
 };
 static const char attrs[][12] = {
-	"action", "archive",
-	"background",
+	"action", "archive",						// Add new entry in sorted position
+	"background",								// as binary search is performed on attrs[][]
 	"code", "codebase", "cite", "classid",
```

Comment:
Age old problem of tabs vs spaces! :) I will use spaces whenever possible.

And about those comments. I lost well over an hour when I added `style` attribute in the end!